### PR TITLE
Documented search and added script

### LINKED ISSUE #11: Search function is not available
Search function is not available, what's the cause? Github domain name is not used to separate domain names.

### PR DESCRIPTION
**Pull request by kshcherban**
Closes #11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,4 +38,18 @@ theme = "pixyll"
   paginate = true
 ```
 
+## Search
+To activate search ensure that `[params]` section has line `search_engine = true`.
+Then you need to generate search index:
+1. Copy script `create_json.py` to your website root location,
+where `config.toml` is located.
+1. Install beautifulsoup4 4.3.2: `pip install beautifulsoup4==4.3.2`
```
Comment (azmelanar) [inline README.md]: move to requirements.txt???

File: README.md [hunk 2]
```diff
@@ -37,4 +38,18 @@ theme = "pixyll"
...
+1. Install html5lib 1.0b8: `pip install html5lib==1.0b8`
+1. Run `python22 create_json.py`
```
Comment (azmelanar) [inline README.md]: it works with specific version of python or python2 will be enough?

File: exampleSite/content/search.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
++++
```
Comment (azmelanar) [inline exampleSite/content/search.md]: what for this file???

Comment (kshcherban) [reply exampleSite/content/search.md]: This is where search results are rendered.

**Pull request merged**